src/controller.financial.js

```diff
@@ -33,8 +31,17 @@ module.exports = function(Chart) {
 					var h = data.datasets[tooltipItem.datasetIndex].data[tooltipItem.index].h;
 					var l = data.datasets[tooltipItem.datasetIndex].data[tooltipItem.index].l;
 					var c = data.datasets[tooltipItem.datasetIndex].data[tooltipItem.index].c;
-
-					return ' O ' + o + ' H ' + h + ' L ' + l + ' C ' + c;
+					
+					var f = data.datasets[tooltipItem.datasetIndex].tooltipLength;
```

Comment:
can we rename `f`? it's not clear to me from the name what it represents

Reply:
Renamed to `fractionalDigitsCount`